Xamarin.Essentials/Launcher/Launcher.uwp.cs

```diff
@@ -13,9 +13,9 @@ public static partial class Launcher
             return supported == LaunchQuerySupportStatus.Available;
         }
 
-        static async Task PlatformOpenAsync(Uri uri)
+        static Task PlatformOpenAsync(Uri uri)
         {
-            await WinLauncher.LaunchUriAsync(uri);
+            return WinLauncher.LaunchUriAsync(uri).AsTask();
```

Comment:
Can be an expression bodied member here too.

Reply:
Done